unity/Assets/Scripts/AgentManager.cs

```diff
@@ -442,6 +444,21 @@ private enum serverTypes { WSGI, FIFO };
             }
         }
 
+        //updates camera near and far clipping planes
+        //default to near and far clipping planes of agent camera, which are currently
+        //static values and are not exposed in anything like Initialize
+        if(nearClippingPlane == null) {
+            camera.nearClipPlane = 0.1f;
+        }
+
+        else { camera.nearClipPlane = (float) nearClippingPlane;}
+
+        if(farClippingPlane == null) {
+            camera.farClipPlane = 20.0f;
+        }
+
+        else { camera.farClipPlane = (float) farClippingPlane;}
```

Comment:
Wait, one other bug (appears twice, also for `nearClipPlane`)... If these are `null` (from UpdateThirdPartyCameras), then they should be keeping their current values.

Can
```c#
        if (farClippingPlane == null) {
            camera.farClipPlane = 20.0f;
        } else {
            camera.farClipPlane = (float) farClippingPlane;
        }
```

then be changed to
```c#
        if (farClippingPlane != null) {
            camera.farClipPlane = (float) farClippingPlane;
        }
```

Reply:
gotchya, changed with latest commit. Also formatted all changes (Format Selection works flawlessly)